projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/bulk-pricing/b2b-bulk-pricing.e2e-spec.ts

```diff
@@ -0,0 +1,24 @@
+import * as bulkPricingActions from '../../../../helpers/b2b/b2b-bulk-pricing';
+
+describe('B2B - Bulk Pricing', () => {
+  before(() => {
+    cy.window().then((win) => win.sessionStorage.clear());
+  });
+
+  describe('Check bulk pricing table', () => {
+    it('should render pricing table for products that contain bulk prices', () => {
+      bulkPricingActions.visitProductWithBulkPrices();
```

Comment:
Usining helpers here makes those tests very hard to reason about (and fix, if needed) as it's hiding the actual flow under some abstraction. I would advice to inline those helpers here.

Reply:
done